recipe/bld.bat

```diff
@@ -1,2 +1,4 @@
+set "R_CUSTOM_TOOLS_PATH=%BUILD_PREFIX:\=/%/Library/usr/bin2"
```

Comment:
I don't think `R_CUSTOM_TOOLS_PATH` is usable anymore for the current `r-base` builds due to
```diff
# diff -u r-base-4.4.1-hc390f20_{9,10}/lib/R/library/base/R/Rprofile
--- r-base-4.4.1-hc390f20_9/lib/R/library/base/R/Rprofile	2024-07-01 06:53:24.799186944 +0000
+++ r-base-4.4.1-hc390f20_10/lib/R/library/base/R/Rprofile	2024-07-02 04:33:31.126751661 +0000
@@ -194,7 +194,7 @@
 local({
     # keep in step with Rcmd_environ
     setRtools44Path <- 0
-     setRtools44Path <- 1
+    # INSTALLER-BUILD: setRtools44Path <- 1
 
     if (setRtools44Path) {
         aarch64 <- FALSE
```